idx/memory/meta_tags.go

```diff
@@ -75,9 +75,8 @@ func (m metaTagRecords) upsert(record tagquery.MetaTagRecord) (recordId, *tagque
 
 // hashMetaTagRecord generates a hash of all the queries in the record
 func (m *metaTagRecords) hashMetaTagRecord(record tagquery.MetaTagRecord) recordId {
```

Comment:
why is this a method of MetaTagRecords and not of tagquery.MetaTagRecord ?

Reply:
good point, that would be better